plugins/out_s3/s3.c

```diff
@@ -1531,8 +1533,8 @@ static struct multipart_upload *create_upload(struct flb_s3 *ctx,
         flb_errno();
         return NULL;
     }
-    s3_key = flb_get_s3_key(ctx->s3_key_format, time(NULL), tag, ctx->tag_delimiters,
-                            ctx->seq_index);
+    s3_key = flb_get_s3_key(ctx->s3_key_format, file_first_log_time, tag,
```

Comment:
So this covers multipart, but I believe there is another part of the code base where we call `flb_get_s3_key` for PutObject. I do not see that change.